File: solutions/1200-1299/1295_find-numbers-with-even-number-of-digits/solution.go
package leetcode

func find_count(num int) int {
	var count int = 0
	for num > 0 {
		num = num / 10
		count++
	}
	return count
}

func findNumbers(nums []int) int {
	var res int = 0
	for i := 0; i < len(nums); i++ {
		if find_count(nums[i])%2 == 0 {
			res++
		}
	}
	return res
}

File: solutions/1200-1299/1295_find-numbers-with-even-number-of-digits/solution.java
class Solution {
    public int findCount(int num){
        int count = 0;
        while (num > 0){
            num = num/10;
            count++;
        };
        return count;
    }
    public int findNumbers(int[] nums) {
        int res = 0;
        for (int i = 0; i < nums.length; i++){
            if (findCount(nums[i])%2==0){
                res++;
            }
        }
        return res;
    }
}